benchmark/gcbench/vdparser.extra/vdc/ast/type.d

```diff
@@ -906,7 +906,7 @@ class TypeDynamicArray : TypeIndirection
         if(super.convertableFrom(from, flags))
             return true;
 
-        if(from.classinfo == typeid(TypeStaticArray))
+        if (typeid(from) is typeid(TypeStaticArray))
```

Comment:
Another tiny change I hope is welcome: `==` to `is`, as is done everywhere else in the code.